src/PatchOrchestrationApplication/CoordinatorService/src/CoordinatorService.cs

```diff
@@ -113,6 +112,10 @@ private async Task RunLoopAsync(CancellationToken cancellationToken)
                     await this.rmHelper.TimeoutRepairTasks(cancellationToken);
                     // Cleanup the ResultStore in case it exceeds the quota
                     await this.CleanupWuOperationResult(cancellationToken);
+                    // This task will post updates of Repair tasks on the Coordinator Service.
+                    await this.rmHelper.PostClusterPatchingStatus(cancellationToken);
+                    // Clears the orphan event posted on coordinator service.
```

Comment:
So the changes to the json/ file format are not part of the PR?

Reply:
yes